packages/select/src/components/select/_select.scss

```
@@ -3,26 +3,32 @@
 
 @import "../../common/variables";
 
+$select-padding: $pt-grid-size * 0.5;
+
 .#{$ns}-select-popover {
   .#{$ns}-popover-content {
     // use padding on container rather than margin on input group
     // because top margin leaves some empty space with no background color.
-    padding: $pt-grid-size * 0.5;
+    padding: $select-padding;
   }
 
   .#{$ns}-input-group {
     margin-bottom: 0;
   }
 
   .#{$ns}-menu {
+    // offset the padding added by the select componenent so menu can remain full width with padding on either side
+    // for menu items
+    margin-left: -$select-padding;
+    margin-right: -$select-padding;
     max-height: $select-popover-max-height;
     max-width: $select-popover-max-width;
     overflow: auto;
-    padding: 0;
+    padding: 0 5px;
```

nitpick: Does it make sense for this to also be `$select-padding`?

```suggestion
    padding: 0 $select-padding;
```

---

indeed!